libs/composer/vendor/composer/ClassLoader.php

```diff
@@ -272,6 +273,26 @@ public function isClassMapAuthoritative()
     }
 
     /**
+     * APCu prefix to use to cache found/not-found classes, if the extension is enabled.
```

Comment:
These changes are only related to a newer composer-version which produced the code